packages/block-library/src/blocks/sidecar/scss/_block-content-alignments.scss

```diff
@@ -0,0 +1,336 @@
+      }
+      
+      &[class*='stop'] {
+        --block-content-start: cs;
+        --block-content-end: var(--block-limit-right);
+      }
+    }
+  }
+}
+
+// TWO SIDEBARS
+.novablocks-sidecar--complex {
+  
+  .novablocks-content:not(:only-child)[class] {
+  
+    .pull-left {
+  
+      @include grid-item-position('left');
+  
+      // There is not space on the left side,
+      // so we are going to treat pull left as pull right.
+      &.stop-left {
+        @include grid-item-position('right');
+      }
+    }
+  
+    .pull-right {
+  
+      @include grid-item-position('right');
+    
+      // There is not space on the left side,
+      // so we are going to treat pull left as pull right.
+      &.stop-right {
+        @include grid-item-position('left');
+      }
+    }
+    
+    .alignleft {
+  
+      @include grid-item-position('left');
+      
+      &[class*='stop-left'] {
+        @include grid-item-position('right');
+      }
+      
+      &[class*='stop-right'] {
+        @include grid-item-position('left');
+      }
+    }
+    
+    .alignright {
+      
+      &[class*='stop'] {
+        @include grid-item-position('content-right');
+    
+        + *:not([class*='align']) {
+          --block-content-end: gcs;
+        }
+      }
+    }
+    
+    .alignleft,
+    .alignright,
+    .pull-left,
+    .pull-right {
+    
+      + *:not([class*='align']) {
+        
+        --block-content-start: cs;
+      }
+    
+    }
+  
+  }
+}
+
+
+// THIS WILL BE AVAILABLE IN
+// THE NEXT VERSIONS OF NOVA BLOCKS.
+// SIDECAR IN SIDECAR
+// On Hold for the moment
+//.novablocks-sidecar:not(.ignore-block) {
+//
+//  // Sidecar with Sidebar Left
+//  // inside Sidecar with Sidebar Right
+//  .novablocks-sidecar {
+//
+//    .novablocks-content {
+//
+//      @include grid-item-position('content');
+//
+//      .pull-left {
+//        @include grid-item-position('left');
+//      }
+//
+//      .pull-right {
+//        @include grid-item-position('right');
+//      }
+//
+//      .alignleft {
+//
+//        @include grid-item-position('left');
+//
+//        &[class*='stop'] {
+//          @include grid-item-position('right');
+//        }
+//      }
+//
+//      .alignright {
+//
+//        @include grid-item-position('right');
+//
+//        &[class*='stop'] {
+//          @include grid-item-position('left');
+//        }
+//      }
+//
+//      //.alignleft,
+//      //.alignright,
+//      //.pull-left,
+//      //.pull-right {
+//      //
+//      //  + *:not([class*='align']) {
+//      //    --block-content-start: cs;
+//      //    --block-content-end: ce;
+//      //  }
+//      //}
+//    }
+//  }
+//
+//  // Sidecar with Sidebar Right
+//  // inside Sidecar with Sidebar Left
+//
+//  &.novablocks-sidecar--sidebar-left {
+//
+//    .novablocks-sidecar {
+//
+//      .novablocks-sidebar {
+//
+//        grid-column: ge/var(--block-limit-right);
+//      }
+//    }
+//  }
+//}
```

Comment:
Is this still relevant?

Reply:
Not for this release. Sidecar in Sidecar and layout with two sidebars is going to be released in the next version probably. I thought it would be easier to have the code here for the next update.